OpenKh.Research.Panacea/OpenKh.Research.Panacea.vcxproj

```diff
@@ -87,6 +89,9 @@
       <GenerateDebugInformation>true</GenerateDebugInformation>
       <EnableUAC>false</EnableUAC>
       <AdditionalDependencies>Shlwapi.lib;Dependencies\bass.lib;Dependencies\bass_vgmstream.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalLibraryDirectories>
+      </AdditionalLibraryDirectories>
```

Comment:
can confirm these two lines above are not needed.

Reply:
lmfao let them stay, C++ is weird.

Reply:
but it'll bother me.

Reply:
Alright then. You are responsible if it breaks :I

Reply:
oh. it wont I tested it before saying it was unneeded.